spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -87,6 +88,9 @@
 
 	private final TaskSanitizer taskSanitizer = new TaskSanitizer();
 
+	private static final List<String> allowedSorts = Arrays.asList("TASK_EXECUTION_ID", "TASK_NAME", "START_TIME",
+			"END_TIME", "EXIT_CODE");
```

Comment:
In SCT, I included the ability to sort by any field in the table...Are these the only fields we ever allow the user to sort by in the UI?

Reply:
Yeah, I took those from there. If there's other fields which work we could add those as well.